geolocation/src/web.ts

```diff
@@ -0,0 +1,82 @@
+import { WebPlugin } from '@capacitor/core';
+
+import type {
+  GeolocationPlugin,
+  GeolocationOptions,
+  GeolocationPosition,
+  GeolocationWatchCallback,
+  GeolocationPermissionStatus,
+} from './definitions';
+import { extend } from './util';
+
+export class GeolocationWeb extends WebPlugin implements GeolocationPlugin {
+  async getCurrentPosition(
+    options?: GeolocationOptions,
+  ): Promise<GeolocationPosition> {
+    return new Promise((resolve, reject) => {
+      navigator.geolocation.getCurrentPosition(
+        pos => {
+          resolve(pos);
+        },
+        err => {
+          reject(err);
+        },
+        extend(
+          {
+            enableHighAccuracy: true,
+            timeout: 10000,
+            maximumAge: 0,
+          },
+          options,
+        ),
```

Comment:
These should be equivalent, and then you can delete the `extend` utility.

```suggestion
        {
          enableHighAccuracy: true,
          timeout: 10000,
          maximumAge: 0,
          ...options,
        },
```